osal/win32/osal.c

```diff
@@ -48,6 +48,24 @@ static double qpc2usec;
 
 #define USECS_PER_SEC     1000000
 
+int osal_getabsolutetimeofday(struct timeval *tv, struct timezone *tz)
```

Comment:
How does this function get affected by system time?  For the stack the most important task is to work as a stop watch and monitor when we have waited or tried long enough. It is kind of legacy from Linux that it is named "gettimeofday".